public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js

```diff
@@ -204,7 +204,8 @@ class DefineMonitor extends Component {
         _.set(monitor, 'triggers', []);
 
         switch (searchType) {
-          case SEARCH_TYPE.QUERY || SEARCH_TYPE.GRAPH:
+          case SEARCH_TYPE.QUERY:
```

Comment:
Should there be something defined here?

Reply:
My understanding is that [javascript's `switch` syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) doesn't use `||` to indicate `or`. You instead list the cases one after the other, followed by the action, to indicate that the function should perform that action in all of those cases. So in this context, the function will execute line 209 in cases where `searchType` is `GRAPH` or `QUERY`.